peps/pep-0013.rst

```diff
@@ -357,6 +357,7 @@ History of amendments
 * 2024-12-10: `Adopted
   <https://discuss.python.org/t/changing-pep-13-to-adopt-bloc-star-voting/64971>`__
   Multi-winner Bloc STAR voting for council elections.
+* 2024-12-10: Added a one-week deadline for seconding a vote of no confidence.
```

Comment:
Updated to resolve merge conflict after STAR voting was added in https://github.com/python/peps/pull/3958.

Which do we prefer, link the date?

```suggestion
* `2024-12-10 <https://discuss.python.org/t/64971>`__:
  Adopted Multi-winner Bloc STAR voting for council elections.
* `2024-12-10 <https://discuss.python.org/t/72293/4>`__:
  Added a one-week deadline for seconding a vote of no confidence.
```

Or link the first word?

```suggestion
* 2024-12-10: `Adopted
  <https://discuss.python.org/t/64971>`__
  Multi-winner Bloc STAR voting for council elections.
* 2024-12-10: `Added
  <https://discuss.python.org/t/72293/4>`__
  a one-week deadline for seconding a vote of no confidence.
```

Reply:
I prefer the date, since it's something that's more likely to be consistent across all the updates + the markup wraps nicely as well.

But also... they both work well-enough that it should be good. :)